### PR TITLE
Add PT37 source code in the repo

### PR DESCRIPTION
**Pull request by lisaleejz**
*Issue #, if available:*

*Description of changes:*
Originally the source code is downloaded when following instructions in the Jupyter notebook,
commit the source code here for tracking changes

dependencies - the docker image as the environment to run the app
packages/src/yolov5 - the model source code (removed, only keep one photo for mock frame use)
packages/src/yolov5s_model - the built yolov5s model
packages/src/app - panorama application source code

also rename the package as **yolov5s_p37_app**

yolov5s fp16 pytorch (torch-1.10.0) jit model is provided.

_By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice._

### CODE REVIEW

File: samples/PT37_opengpu/yolov5s_pt37_app/packages/028663699634-yolov5s_pt37_app-1.0/src/app/app_fp16.py [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+                total_process_metric = self.metrics_handler.get_metric('TotalProcessBatchTime')
+                input_images_batch = image_list[:self.model_batch_size]
+
+                preprocessing_metric = self.metrics_handler.get_metric('PreProcessBatchTime')
+                pre_processed_images = [torch.from_numpy(img_utils.preprocess_v2(image)).to(self.device).half() for image in input_images_batch]
+                preprocessing_metric.add_time_as_milliseconds(1)
+                self.metrics_handler.put_metric(preprocessing_metric)
+
+                # Create Torch Stack
+                pre_processed_images = torch.stack(pre_processed_images) # 4 (batch size) * [1 * 100 * 100] -> [4, 1, 100, 100]
```
Comment (kevhsu-k) [inline samples/PT37_opengpu/yolov5s_pt37_app/packages/028663699634-yolov5s_pt37_app-1.0/src/app/app_fp16.py:101]: Shall we put it inside preprocessing?

File: samples/PT37_opengpu/dependencies/README.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# Build instructions
```
Comment (kevhsu-k) [inline samples/PT37_opengpu/dependencies/README.md:1]: I guess we can remove this Readme.md

**Comment by kevhsu-k** [comment]
Maybe we don't need yolov5s.pt anymore. As we are using torchscript fp16 model now.

File: samples/PT37_opengpu/yolov5s_pt37_app/packages/028663699634-yolov5s_pt37_app-1.0/src/app/s3_utils.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import boto3
+s3 = boto3.resource('s3')
+
+def send_to_s3(self, image_to_send, bucket_name, filename):
```
Comment (kevhsu-k) [inline samples/PT37_opengpu/yolov5s_pt37_app/packages/028663699634-yolov5s_pt37_app-1.0/src/app/s3_utils.py:4]: Is this file still needed?

**Review by SuryaKari — APPROVED** [review]
Reviewed with Lisa and Kevin. A new PR is expected to make the post processing more efficient. But for now, we are only reporting the Pre and Inference times so this is a good
